alluxio-shaded-hadoop/pom.xml

```diff
@@ -0,0 +1,233 @@
+<!--
+
+    The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+    (the "License"). You may not use this work except in compliance with the License, which is
+    available at www.apache.org/licenses/LICENSE-2.0
+
+    This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied, as more fully set forth in the License.
+
+    See the NOTICE file distributed with this work for information regarding copyright ownership.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.alluxio</groupId>
+    <artifactId>alluxio-parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>alluxio-shaded-hdfs</artifactId>
+  <packaging>jar</packaging>
+  <name>alluxio-shaded-hdfs</name>
+  <version>${ufs.hadoop.version}-2.0.0-SNAPSHOT</version>
+  <description>Alluxio shaded hdfs</description>
+
+  <properties>
+    <!-- The following paths need to be defined here as well as in the parent pom so that mvn can -->
+    <!-- run properly from sub-project directories -->
+    <build.path>${project.parent.basedir}/build</build.path>
+    <ufs.hadoop.version>2.2.0</ufs.hadoop.version>
+    <shading.prefix>alluxio.shaded.hdfs</shading.prefix>
+    <!-- Composite jar name with both hadoop version and project version included -->
+    <lib.jar.name>${project.artifactId}-${ufs.hadoop.version}-${project.version}.jar</lib.jar.name>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-log4j12</artifactId>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+    <!-- using older version to match dependency version from Hadoop -->
+    <groupId>com.google.guava</groupId>
+    <artifactId>guava</artifactId>
+    <version>14.0.1</version>
+  </dependency>
+  </dependencies>
+
+  <profiles>
```

Comment:
can we remove these profiles?

Reply:
i think we need these profiles to include the different hadoop artifacts for different hadoop versions.

Reply:
Ah that makes sense. I think we can get rid of the igormaznitsa jcp parts though

Reply:
done